src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,32 @@
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { TodoContext } from '../TodoContext';
 
 export const TodoModal: React.FC = () => {
+  const [showedUser, setShowedUser] = useState<User | null>(null);
+  const { showedTodo, setShowedTodo } = useContext(TodoContext);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    if (showedTodo?.userId) {
+      getUser(showedTodo.userId)
+        .then(setShowedUser)
+        .catch((error) => {
+          // eslint-disable-next-line no-console
+          console.error(error);
+        })
+        .finally(() => setIsLoading(false));
+    }
+  }, []);
+
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
+      {isLoading && !showedUser ? (
```

Comment:
```suggestion
      {isLoading ? (
```
It's better to check if it's loading, so the user won't be confused about why the user stays the same while loading